pkgs/applications/snowmachine/default.nix

```
@@ -0,0 +1,21 @@
+{ buildPythonPackage, lib, click, colorama, fetchPypi, setuptools-git }:
+
+buildPythonPackage rec {
+  pname = "snowmachine";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1v385hhxy2a8vx5p0fhn0di8l4qfpb0a86j6nwsg0aw6ngb09qf1";
+  };
+
+  buildInputs = [ setuptools-git ];
+  propagatedBuildInputs = [ click colorama ];
+
```

```suggestion
  doCheck = false;
```

Does the package contain tests? Can we do an pythonImportsCheck?

---

The package does not contain any tests. I added `pythonImportsCheck`:
```diff
commit ae0e0d21cfeaf977b38a316988f23330c948d882
Author: Jonathan Strickland <djanatyn@gmail.com>
Date:   Sun Jan 17 16:06:20 2021 -0500

    snowmachine: add pythonImportsCheck
    
    Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

diff --git a/pkgs/applications/snowmachine/default.nix b/pkgs/applications/snowmachine/default.nix
index d34f46d7d0d..01c8d050fdc 100644
--- a/pkgs/applications/snowmachine/default.nix
+++ b/pkgs/applications/snowmachine/default.nix
@@ -12,6 +12,9 @@ buildPythonPackage rec {
   buildInputs = [ setuptools-git ];
   propagatedBuildInputs = [ click colorama ];
 
+  doCheck = false;
+  pythonImportsCheck = [ "click" "colorama" ];
+
   meta = with lib; {
     description = "A python script that will make your terminal snow";
     homepage = "http://github.com/sontek/snowmachine";

```